lua/packer.lua

```diff
@@ -123,6 +126,9 @@ packer.init = function(user_config)
 end
 
 packer.make_commands = function()
+  vim.cmd [[command! -nargs=+ PackerSnapshot  lua require('packer').snapshot(<q-args>)]]
```

Comment:
I could be wrong, but I think we want to use `<f-args>` instead of `<q-args>` here (and for the other snapshot commands)?